erpnext/accounts/doctype/accounts_settings/accounts_settings.json

```diff
@@ -58,7 +59,6 @@
   "closing_settings_tab",
   "period_closing_settings_section",
   "acc_frozen_upto",
-  "ignore_account_closing_balance",
```

Comment:
@rtdany10 This got removed, please add back

Reply:
Sorry, have fixed